mne/cov.py

```diff
@@ -1252,8 +1252,8 @@ def _auto_low_rank_model(data, mode, n_jobs, method_params, cv,
 class _RegCovariance(BaseEstimator):
     """Aux class."""
 
-    def __init__(self, info, grad=0.1, mag=0.1, eeg=0.1, seeg=0.1, ecog=0.1,
-                 hbo=0.1, hbr=0.1, fnirs_cw_amplitude=0.1,
+    def __init__(self, info, grad=0.1, mag=0.1, eeg=0.1, seeg=0.1, dbs=0.1,
+                 ecog=0.1, hbo=0.1, hbr=0.1, fnirs_cw_amplitude=0.1,
                  fnirs_fd_ac_amplitude=0.1, fnirs_fd_phase=0.1, fnirs_od=0.1,
                  csd=0.1, store_precision=False, assume_centered=False):
```

Comment:
same here. You cannot insert a parameter in a list without breaking API unless you put it at the end or you enforce named parameters.